### PR TITLE
Vote style

### PR DESCRIPTION
**Pull request by Haroenv**

### CODE REVIEW

File: _components/poll-vote.md [hunk 1]
```diff
@@ -6,25 +6,45 @@ layout: component
 Once a poll is created, you can vote on it like this:
 
 ```html
-<div class="poll--vote">
-  <label for="optionXX">Some option :)</label>
-  <input type="checkbox" name="optionXX" id="optionXX">
-</div>
+<div class="vote--options">
```
Comment (Haroenv) [inline _components/poll-vote.md]: this should be container

File: _components/poll-vote.md [hunk 2]
```diff
@@ -6,25 +6,45 @@ layout: component
...
+  
+  <div class="vote--options--container">
```
Comment (Haroenv) [inline _components/poll-vote.md]: this should be item

File: _sass/components/vote.scss [hunk 3]
```diff
@@ -1,21 +1,68 @@
...
+		label::before {
+			left: 0;
+		  border: $s-hair solid $c-border-input-light;
+		  box-shadow: inset $s-hair $s-hair rgba(0, 0, 0, 0.03);
+		  border-radius: 50%;
+		}
+
+		label::after {
+			left: 3px;
+		  background: url("https://s3-us-west-2.amazonaws.com/s.cdpn.io/9487/icon-check.svg") no-repeat center center;
```
Comment (Haroenv) [inline _sass/components/vote.scss]: here too, should be the real svg

File: _sass/components/vote.scss [hunk 4]
```diff
@@ -1,30 +1,77 @@
...
+		label::before {
+			left: 0;
+			border: $s-hair solid $c-border-input-light;
+			box-shadow: inset $s-hair $s-hair rgba(0, 0, 0, 0.03);
+			border-radius: 50%;
+		}
+
+		label::after {
+			left: 3px;
+			background: url("https://s3-us-west-2.amazonaws.com/s.cdpn.io/9487/icon-check.svg") no-repeat center center;
```
Comment (Haroenv) [inline _sass/components/vote.scss]:
```css
background: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path fill="none" stroke="#5D8CE7" stroke-width="3" stroke-miterlimit="10" d="M2 6l4 4 8-8"/></svg>') no-repeat center center;
```

**Review by Haroenv — COMMENTED** [review]
great!

**Review by thysruben — APPROVED** [review]
👌

**Pull request merged**